### ISSUE TITLE
Authorization using new Instagram API Scopes

### ISSUE THREAD

**Issue by dzotokan** [prowdsponsor/ig#22]
Hi there,

I've been trying to use `getRecentTagged` which relies on auth token which itself depends on `getUserAccessTokenURL1`.

The response I get is:

```
*** Exception: IGAppException (IGError {
  igeCode = 400, 
  igeType = Just "OAuthPermissionsException", 
  igeMessage = Just "This request requires scope=public_content, but this access token is not authorized with this scope. The user must re-authorize your application with scope=public_content to be granted this permissions."})
```

Started looking at [`Scope`](https://hackage.haskell.org/package/ig-0.6.1/docs/src/Instagram-Types.html#Scope) definition in the library and figured there is no constructor for `public_content` and `follower_list` [Instagram API | Scopes](https://www.instagram.com/developer/authorization/).

Any chance new scopes can be added in the nearest future? :pray:

**Comment by meteficha** [comment]
Currently I'm not adding new features to the library.  Can you add the scope and a unit test then send a PR?

**Comment by dzotokan** [comment]
Thanks for the quick response! PR is up - would be great to get it looked at at some point. :eyes:

**Comment by cdepillabout** [comment]
Closed by #23.

**Comment by dzotokan** [comment]
Fantastic, thanks!
